spinalcordtoolbox/deepseg/models.py

```diff
@@ -120,33 +122,105 @@
+                             'subjects in total: 49 with tumors of type Astrocytoma, 83 with Ependymoma, and 111 with '
+                             'Hemangioblastoma. For each subject, the requisite parts of the affected region (tumor, '
+                             'edema, cavity) were segmented individually for training purposes. This model is used in '
+                             'tandem with another model for specialized cord localisation of spinal cords with tumors '
+                             '(https://github.com/ivadomed/findcord_tumor).',
+         'url': 'https://github.com/ivadomed/model_seg_sctumor-edema-cavity_t2-t1_unet3d-multichannel',
          'models': ['findcord_tumor', 'model_seg_sctumor-edema-cavity_t2-t1_unet3d-multichannel']},
     'seg_exvivo_gm-wm_t2':
         {'description': 'Grey/white matter seg on exvivo human T2w',
+         'long_description': 'This segmentation model for T2w human spinal gray and white matter uses a 2D Unet '
+                             'architecture, and was created with the `ivadomed` package. Training data consisted '
+                             'of 149 2D slices taken from the scan of an ex vivo spinal cord of a single subject, with '
+                             'the gray and white matter from each slice manually segmented for training purposes. The'
+                             'data was provided by the University of Queenland, and through this collaboration, the '
+                             'model was subsequently applied to the development of an ex vivo spinal cord template '
+                             '(https://archive.ismrm.org/2020/1171.html).',
+         'url': 'https://github.com/ivadomed/model_seg_exvivo_gm-wm_t2_unet2d-multichannel-softseg',
          'models': ['model_seg_exvivo_gm-wm_t2_unet2d-multichannel-softseg']},
     'seg_gm_sc_7t_t2star':
-        {'description': 'SC/GM seg on T2*-weighted at 7T',
+        {'description': 'SC/GM seg on T2*-weighted contrast at 7T',
+         'long_description': 'This multiclass model (SC/GM) has been developed from 72 subjects acquired at 7T '
+                             '(T2*-w images of the cervical spinal cord) by N.J. Laines Medina, V. Callot and '
+                             'A. Le Troter in the Center for Magnetic Resonance in Biology and Medicine '
+                             '(CRMBM-CEMEREM, UMR 7339, CNRS, Aix-Marseille University, France) including various '
+                             'pathologies (HC, MS, ALS). It was validated between 9-folds (CV) single-class and '
+                             'multi-class models and was enriched by integrating a hybrid data augmentation (composed '
+                             'of classical geometric transformations, MRI artifacts and real GM/WM contrasts distorted '
+                             'with anatomically constrained deformation fields) finally tested with an external '
+                             'multicentric database. For more information visit: ',
```

Comment:
I'm pretty sure this is a run-on sentence. Here's a quick attempt at making it more grammatical / easier to read, hopefully without distorting the meaning:
> This multiclass model (SC/GM) was developed by N.J. Laines Medina, V. Callot and A. Le Troter at the Center for Magnetic Resonance in Biology and Medicine (CRMBM-CEMEREM, UMR 7339, CNRS, Aix-Marseille University, France). Training data consisted of T2*w scans acquired at 7T from 72 subjects: 34 healthy controls, 25 patients with ALS, 13 patients with MS. The model was validated by comparing with single-class models using 9-fold Cross-Validation. It was enriched by integrating a hybrid data augmentation (composed of classical geometric transformations, MRI artifacts, and real GM/WM contrasts distorted with anatomically constrained deformation fields). Finally, it was tested with an external multicentric database. For more information, see the following URL.

(I got the subject numbers from [the linked paper on arXiv](https://arxiv.org/abs/2110.06516))